{
  "comments": [
    {
      "key": {
        "uuid": "da86d52c_88a3aa4e",
        "filename": "tests/test_jenkins.py",
        "patchSetId": 1
      },
      "lineNbr": 11,
      "author": {
        "id": 679
      },
      "writtenOn": "2015-02-06T22:08:10Z",
      "side": 1,
      "message": "This should be on line 10, with line 11 being blankâ€”i.e., second set of imports rather than third set.",
      "revId": "456bcfb4f9c02f9910df1360f9163690106d3225",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543",
      "unresolved": false
    }
  ]
}